src/core/lightning/index.ts

```diff
@@ -58,8 +53,11 @@ export const LightningMixin = (superclass) =>
           lock,
           logger: this.logger,
         }),
-        this.updatePendingPayments(lock),
-        super.updatePending(lock),
+        Wallets.updatePendingPayments({
```

Comment:
Note: this method can return a `LedgerServiceError` but in instances like this we don't check for it and throw. Is this ok?

Reply:
When this is being substituted in the old code we should throw if its an error to mirror existing behaviour

Reply:
I guess we need to throw an exception, or graphql server do the translation? this can break mobile app behavior. @jrbuhl93 could you confirm how do we handle exceptions in get balance?

Reply:
Eseentially we get the balance as part of [this query](https://github.com/GaloyMoney/galoy-mobile/blob/2755d29462fdb76b37698efc2cf3ee63b636fc0d/app/graphql/query.ts#L15). If the query doesn't return a new balance, it should continue to just use the cached balance.

Reply:
Decided to `throw` to be consistent with the old implementation: https://github.com/GaloyMoney/galoy/pull/526/commits/23705555584cea6bdcf0b924e491641ed2a8fe3e